plugins/inputs/sqlserver/README.md

```diff
@@ -56,6 +61,27 @@ GO
   # azuredb = false
 
   ## Possible queries
+  ## Queries enabled by default for specific Database Type
+  
+  ## database_type =  AzureSQLDB
+  ## - AzureDBWaitStats
+  ## - AzureDBResourceStats 
+  ## - AzureDBResourceGovernance
+  ## - AzureDBDatabaseIO
+  ## - AzureDBServerProperties
+  ## - AzureDBSQLOsWaitstats
+  ## - AzureDBMemoryClerks
+  ## - AzureDBPerformanceCounters
+
+   ## database_type =  AzureSQLManagedInstance
```

Comment:
there is an extra space in the query list

Reply:
Changed